app/views/courses/about.html.erb

```diff
@@ -0,0 +1,86 @@
+        <li>information about the prestige of the course (eg league-table rankings)</li>
+      </ul>
+
+      <p class="govuk-body">If you offer different courses in the same subject, be clear about how this course differs from the others (eg amount of time spent in school, course structure). This will help the applicant decide which course best suits them.</p>
+
+      <div class="govuk-character-count" data-module="character-count" data-maxwords="400">
+        <div class="govuk-form-group">
+          <%= f.label :about_course, 'About this course', class: 'govuk-label govuk-label--s' %>
+          <%= f.text_area :about_course, rows: 15, class: 'govuk-textarea js-character-count' %>
+        </div>
+      </div>
+
+      <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl">
+
+      <h3 class="govuk-heading-l remove-top-margin">Interview process</h3>
+
+      <p class="govuk-body">Give applicants a summary of the interview process.</p>
+
+      <p class="govuk-body">This could include:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>who’ll be interviewing them</li>
+        <li>how many interviews they’ll need to attend</li>
+        <li>details of any tests they’ll need to sit</li>
+      </ul>
+
+      <div class="govuk-character-count" data-module="character-count" data-maxwords="250">
+        <div class="govuk-form-group">
+          <%= f.label :interview_process, 'Interview process (optional)', class: 'govuk-label govuk-label--s' %>
+          <%= f.text_area :interview_process, rows: 10, class: 'govuk-textarea js-character-count' %>
+        </div>
+      </div>
+
+      <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl">
+
+      <h3 class="govuk-heading-l remove-top-margin">How school placements work</h3>
+
+      <p class="govuk-body">Tell applicants more about the schools they’ll be teaching in.</p>
+
+      <p class="govuk-body">You could include:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>how the candidate’s schools are selected</li>
+        <li>the average distance candidates have to travel from home to school</li>
+        <li>the age ranges they’ll be teaching (eg 11 to 16 or 11 to 18)</li>
+        <li>how many schools you partner with in total</li>
+        <li>the number of placements a candidate will have</li>
+        <li>how much time candidates spend in each school</li>
+      </ul>
+
+      <div class="govuk-character-count" data-module="character-count" data-maxwords="350">
+        <div class="govuk-form-group">
+          <%= f.label :how_school_placements_work, 'How school placements work', class: 'govuk-label govuk-label--s' %>
+          <%= f.text_area :how_school_placements_work, rows: 10, class: 'govuk-textarea js-character-count' %>
+        </div>
+      </div>
+
+      <%= f.submit "Save", class: "govuk-button", data: { qa: 'course__save' } %>
```

Comment:
The data-qa here isn't being referenced in the tests, is it?

Reply:
Not yet but it will be when the "save" is working :)